src/components/TradeWidget/index.tsx

```diff
@@ -64,22 +65,40 @@ const TradeWidget: React.FC = () => {
   const { sell: sellTokenSymbol, receive: receiveTokenSymbol } = useParams()
 
   const [sellToken, setSellToken] = useState(
-    () => getToken('symbol', sellTokenSymbol, tokens) || getToken('symbol', 'DAI', tokens),
+    () => getToken('symbol', sellTokenSymbol, tokens) || (getToken('symbol', 'DAI', tokens) as Required<TokenDetails>),
```

Comment:
Shouldn't the cast be applied to the first expression as well?

Reply:
no because the first may not be there - the second, DAI or USDC, is